### ISSUE TITLE
command fail

### ISSUE THREAD

**Issue by Xayzer3314** [ubports/ubports-installer#2874]
**UBports Installer `0.9.7-beta` (exe)**
Environment: `Microsoft Windows 10 专业版 10.0.19044 Windows 10.0.19044 x64 19044 0.0 NodeJS v16.15.0`
Device: [`begonia`](https://github.com/ubports/installer-configs/blob/41ea7e9f76948da347aa285d5d2e9a17f9af9857/v2/devices/begonia.yml) ([Xiaomi Redmi Note 8 Pro](https://devices.ubuntu-touch.io/device/begonia/))
Target OS: Ubuntu Touch
Settings: `{"bootstrap":false,"wipe":true,"channel":"16.04/arm64/android9/stable"}`
OPEN-CUTS run: *N/A* 
Pastebin: https://snip.hxrsh.in/ubi-1666108181406

Basically the same error to closed issue#2424, still not working on my device
https://github.com/ubports/ubports-installer/issues/2424
